src/features/page_view_event/aggregate/index.js

```diff
@@ -67,27 +66,27 @@ export class Aggregate extends AggregateBase {
     this.aggregator.store('measures', 'fe', { value: isBrowserScope ? agentRuntime[CONSTANTS.FBTWL] : 0 })
     this.aggregator.store('measures', 'dc', { value: isBrowserScope ? agentRuntime[CONSTANTS.FBTDC] : 0 })
 
-    var measuresMetrics = this.aggregator.get('measures')
-
-    var measuresQueryString = mapOwn(measuresMetrics, function (metricName, measure) {
-      return '&' + metricName + '=' + measure.params.value
-    }).join('')
-
-    // currently we only have one version of our protocol
-    // in the future we may add more
-    var protocol = '1'
-
-    var scheduler = new HarvestScheduler('page_view_event', {}, this)
-
-    var chunksForQueryString = [scheduler.harvest.baseQueryString()]
-
-    chunksForQueryString.push(measuresQueryString)
+    const queryParameters = {
+      tt: info.ttGuid,
+      us: info.user,
+      ac: info.account,
+      pr: info.product,
+      af: getActivatedFeaturesFlags(this.agentIdentifier).join(','),
+      ...(
+        Object.entries(this.aggregator.get('measures') || {}).reduce((aggregator, [metricName, measure]) => {
+          aggregator[metricName] = measure.params?.value
+          return aggregator
+        }, {})
+      ),
+      xx: info.extra,
+      ua: info.userAttributes,
+      at: info.atts
+    }
```

Comment:
```suggestion
    }

    var customJsAttributes = stringify(info.jsAttributes)
    queryParameters.ja = customJsAttributes === '{}' ? null : customJsAttributes
```

Reply:
There's one way to fix the functional tests.

Reply:
The `ja` attribute was moved to the body on purpose.